data/coalition/coalition missions.txt

```diff
@@ -1424,24 +1424,24 @@ mission "Coalition Folklore: Saryd"
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several light rays shine onto it, and as you look up you can only barely see some spotlights high up on the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings on the top. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but once he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks to the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
+			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether. He puts the rod on the boat and tries to steer it back to shore, but he can't find his way. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`
+			`	The fisherman kicks and strugles for air, having lost his boat and still at the mercy of the great waves. The view on the stage starts going down, following the kicks, and something starts rising up to meet the Saryd's legs. As it raises the view back with it, a creature reminiscent of a whale, only with much larger, longer flippers that almost link to its tail, comes to the surface, gently and with care, as if trying to serve as ground for the sailor. Another wave is approaching, large enough to engulf them both, but the whale keeps on going up, until it is out of the water.`
 			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea, and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water, as it dives deep once again, and the curtains close.`
```

Comment:
I feel like it would be more natural to remove a comma there, so sentence flow is better.
```suggestion
			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea, and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water as it dives deep once again, and the curtains close.`
```